config/core/test-configs-chromeos.yaml

```diff
@@ -13,14 +13,14 @@ device_types:
     mach: x86
     arch: x86_64
     boot_method: depthcharge
-    filters: &x86-chromebook-filters
-      - passlist: {defconfig: ['x86-chromebook']}
+    filters: &x86-board-filters
+      - passlist: {defconfig: ['x86-board']}
     params: &octopus-params
       cros_flash_initrd: 'https://storage.staging.kernelci.org/images/rootfs/debian/bullseye-cros-flash/20220523.1/amd64/initrd.cpio.gz'
       cros_flash_nfs: 'https://storage.staging.kernelci.org/images/rootfs/debian/bullseye-cros-flash/20220523.1/amd64/full.rootfs.tar.xz'
       cros_flash_image: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-octopus/20220520.0/amd64/cros-flash.tar.gz'
-      cros_flash_kernel: 'http://storage.chromeos.kernelci.org/images/kernel/v5.10.112-x86-chromebook/kernel/bzImage'
-      cros_flash_modules: 'http://storage.chromeos.kernelci.org/images/kernel/v5.10.112-x86-chromebook/modules.tar.xz'
+      cros_flash_kernel: 'http://storage.chromeos.kernelci.org/images/kernel/v5.10.112-x86-board/kernel/bzImage'
+      cros_flash_modules: 'http://storage.chromeos.kernelci.org/images/kernel/v5.10.112-x86-board/modules.tar.xz'
```

Comment:
I think we need to just empty this file or even delete it, it is not relevant to main instance, as we dont run tast tests.
@gctucker what you think about this?

Reply:
Or maybe simpler, just dont change only these parameters, and i will deal with this file in separate commit

Reply:
Change reverted in the new patch set, thanks for catching it early!